.travis.yml

```diff
@@ -41,9 +40,6 @@ stages:
     - name: Code Quality
       if: type = pull_request
 
-allow_failures:
```

Comment:
If a build fails in a forest and no one is around to hear it, does it make a sound?